ipaserver/install/ipa_cacert_manage.py

```diff
@@ -558,7 +560,8 @@ def prune(self):
         for ca_cert, ca_nickname, _ca_trust_flags in ca_certs:
             if ca_cert.not_valid_after < now:
                 expired_certs.append(ca_nickname)
-                self._delete_by_nickname(ca_nickname, self.options)
+
+        self._delete_by_nickname(expired_certs, self.options)
```

Comment:
Can you add a ```if expired_certs:``` before this? It looks like _delete_by_nickname() will do a bunch of work even if there is nothing to prune:
- pull the list of certs
- create a temporary NSS db
- etc.

Reply:
Good catch, thanks.